RuntimeLicenses/license.txt

```diff
@@ -9,7 +9,8 @@ except to the extent those have additional terms.
 IF YOU COMPLY WITH THESE LICENSE TERMS, YOU HAVE THE RIGHTS BELOW.
 
 1. INSTALLATION AND USE RIGHTS. 
-You may install and use any number of copies of the software to develop and test your applications. 
+You may only use the C# Extension for Visual Studio Code with Visual Studio Code, Visual Studio or Xamarin Studio software to
+help you develop and test your applications.
```

Comment:
But what's the difference between the extension you download and the software as a whole (meaning the source code and the bits you compile from it)? This seems very... sketchy.

I am also not a lawyer, but some quick research leads me to believe they (source code and object code) must indeed be covered by the same license. It's a [Stack Exchange answer](https://opensource.stackexchange.com/questions/2954/do-licenses-actually-apply-to-the-source-code-or-product-itself/3959) but it's enough for me to say this needs to be looked into further.